src/monthComponent.jsx

```diff
@@ -1,27 +1,64 @@
 import React from 'react';
 import DayComponent from './dayComponent.jsx';
+import moment from 'moment';
+
 
 class MonthComponent extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-
+            hovered: false,
+            availability: this.props.dates,
+            monthToRender: this.props.monthToRender,
+            daysInThisMonth : this.props.daysInThisMonth
         };
+
+        this.onMouseOverFn = this.onMouseOverFn.bind(this);
+        this.handlePrevButtonFromCalendarComponent = this.handlePrevButtonFromCalendarComponent.bind(this);
+    }
+
+    onMouseOverFn(e) {
+        e.preventDefault();
+        this.setState({
+            hovered: !this.state.hovered
```

Comment:
consider renaming variable to isHovered?

Reply:
Soooo... I was in the middle of refactoring my components to no longer have the 'onMouseOver' names for these functions. ThenI only had one left to rename...and I went to lunch. 2 hours later, I could not figure out for the life of me why I still had a bug, totally forgetting about the last one. Anywho, appropriate functions were renamed!